drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/widgets/ScenarioGridCell.java

```diff
@@ -20,7 +20,9 @@
 
 public class ScenarioGridCell extends BaseGridCell<String> {
 
+    // true if this cell is already in editing mode
     private boolean isEditing = false;
+    // true if this cell has to show errors
     private boolean isError = false;
```

Comment:
If you rename for example to `isInEditingMode `mode and to `isHavingAnError ` the comments can be left out.